test/e2e/helpers_test.go

```diff
@@ -528,31 +527,48 @@ func verifyIngressWithEchoBackendsInAdminAPI(
 	t.Helper()
 
 	require.Eventually(t, func() bool {
-		d, err := dump.Get(ctx, kongClient, dump.Config{})
+		start := time.Now()
+		defer func() {
+			t.Logf("Try fetching config from %q started at %s, duration %v", kongClient.BaseRootURL(), start.Format(time.RFC3339), time.Since(start))
```

Comment:
This fill log after the fetching is done so I believe the log should reflect that:

```suggestion
			t.Logf("Fetched config from %q, started at %s, duration %v", kongClient.BaseRootURL(), start.Format(time.RFC3339), time.Since(start))
```